tests/e2e/vsan_stretched_cluster.go

```diff
@@ -233,11 +248,16 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		defer func() {
 			deleteService(namespace, client, service)
 		}()
-		ginkgo.By("Creating statefulset and deployment with volumes from the stretched datastore")
-		statefulset, _, _ := createStsDeployment(ctx, client, namespace, sc, true,
-			false, 0, "", "", false)
+
+		statefulset, _, _ = createStsDeployment(ctx, client, namespace, sc, true,
```

Comment:
see if deployment pods and sts pods can read/write data onto a volume and also make sure if you can attach multiple pods to a single rwx PVC and perform reading/writing of data into the volume from different pods

Reply:
Read write volume is handled in "Primary site down" test case